pkg/users/account-create-dialog.js

```diff
@@ -209,9 +211,10 @@ export function account_create_dialog(accounts) {
 
         errors.user_name = validate_username(state.user_name, accounts);
 
-        return password_quality(state.password)
+        return password_quality(state.password, force)
                 .catch(ex => {
-                    errors.password = ex.message || ex.toString();
+                    errors.password = (ex.message || ex.toString()).replace("\n", " ");
+                    errors.password += "\n" + _("Press again to confirm this weak password.");
```

Comment:
Can we be sure that "weak password" is the only exception ever being thrown here? What if pwquality isn't installed/set up correctly/some other OS error happened?

Reply:
well, won't we then allow user to overrule? When pwquality is not installed for example, we surely want user to be able to create user.

Reply:
Yeap.

`sudo mount -o bind /usr/bin/false /usr/bin/pwscore`

and then in the UI `Password is not acceptable` and you cannot proceed.

Reply:
Showing an exception text plus "press again to do it anyway" if pwscore fails feels a little weird -- but at the same time, we do want to inform the user about that failure somehow. Given how badly broken it is right now in the "pwscore fails" case, I think it's good and robust enough now.